### ISSUE TITLE
Custom Point Tracking

### ISSUE THREAD

**Issue by rnag5076** [MagicLeapResearch/SuperPointPretrainedNetwork#17]
How can I use the code to provide custom points for tracking in the whole video? Suppose I want only 4 points to be tracked in the whole video so how can I specify them in the code to customize it.

**Comment by sarlinpe** [comment]
You would need to provide the same 4 keypoints for each frame of the video. If you want to specify only for the first frame, I suggest looking into optical flow or dense matching instead.

**Comment by rnag5076** [comment]
Yes, I want to specify only for the first frame but is it possible to use the points that came as an output for the first frame to be used as an input for the second frame?
